### ISSUE TITLE
cross-platform support?

### ISSUE THREAD

**Issue by Neurrone** [dkager/tolk#5]
Are there any future plans of adding support for other screen readers on other operating systems? The ones that come to mind are voice over (mac/iOS), TalkBack on android and Orca on Linux?

**Comment by dkager** [comment]
It would indeed be the next logical step to take. There would have to be a build system, instead of the current collection of simple batch-files, to build a multi-platform library. Also, I don't have experience with building for either OS X or Linux. Something to investigate when time allows!

**Comment by brightening-eyes** [comment]
it is possible, but it must not be dependent on com which windows screen readers are using it
linux has something which emulate's com, but i haven't seen it

**Comment by dkager** [comment]
This isn't going to happen.

* It would involve some drastic changes.
* The platforms mentioned have only one viable screen reader each, so Tolk's abstraction makes no sense.

**Comment by kyleman** [comment]
This is actually far from the truth. The abstraction should be thought of as abstracting speech and screenreader access period. This means regardless of the platform and screenreader. if you want speech, just call Tolk_Output and it will speak no matter where its running like it says in the readme. 
At this point this is only slightly better than BGT. It allows you to use a main-stream language, but you are still tied to windows.

**Comment by cybik** [comment]
If I may:

@dkager Tolk's abstraction makes sense if you look at it from the higher point of view of implementing once with Tolk, and having it work on W/M/L without modification.
